browser-test/src/applicant/questions/file.test.ts

```diff
@@ -336,6 +336,83 @@ test.describe('file upload applicant flow', () => {
       await validateScreenshot(page, 'file-uploaded-multiple-files')
     })
 
+    test('review page renders correctly', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(programName)
+      await applicantQuestions.answerFileUploadQuestion(
+        'file 1 content',
+        'file1.txt',
+      )
+      await applicantQuestions.answerFileUploadQuestion(
+        'file 2 content',
+        'file2.txt',
+      )
+
+      await applicantQuestions.clickReview()
+
+      await applicantQuestions.expectQuestionAnsweredOnReviewPage(
+        fileUploadQuestionText,
+        'file1.txt',
+      )
+
+      await applicantQuestions.expectQuestionAnsweredOnReviewPage(
+        fileUploadQuestionText,
+        'file2.txt',
+      )
+      await validateScreenshot(page, 'file-uploaded-review')
```

Comment:
**suggestion (non-blocking):** Rather than taking a screenshot of the whole page including the header and footer, could you take a screenshot of the relevant section?  That'll prevent less churn when we update other parts of the page.

Reply:
Done